provider/source/src/datetime/neo_skeleton.rs

```diff
@@ -149,7 +149,8 @@ impl SourceDataProvider {
     ) -> Result<PackedPatterns<'static>, DataError> {
         let data = self.get_datetime_resources(locale, calendar)?;
 
-        let length_combinations_v1 = GenericLengthPatterns::from(&data.datetime_formats);
+        // Note: We default to atTime here (See #469)
+        let length_combinations_v1 = GenericLengthPatterns::from(&data.datetime_formats_at_time);
```

Comment:
Thought: I'm not immediately sure whether the output of this code is reachable from the datetime component. Still good to migrate it.